source/rules/no-ignored-error.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license Use of this source code is governed by an MIT-style license that
+ * can be found in the LICENSE file at https://github.com/cartant/eslint-plugin-rxjs
+ */
+
+import { Rule } from "eslint";
+import * as es from "estree";
+import {
+  getParent,
+  isArrowFunctionExpression,
+  isFunctionDeclaration,
+  typecheck
+} from "../utils";
+
+const rule: Rule.RuleModule = {
+  meta: {
+    docs: {
+      category: "RxJS",
+      description:
+        "Disallows the calling of subscribe without specifying an error handler.",
+      recommended: false
+    },
+    fixable: null,
+    messages: {
+      forbidden: "Calling subscribe without an error handler is forbidden."
+    },
+    schema: []
+  },
+  create: context => {
+    const { couldBeObservable, isReferenceType, couldBeFunction } = typecheck(
+      context
+    );
+
+    function nodeIsLikelyAFunction(node: es.Node): boolean {
+      // Fast check
+      if (isArrowFunctionExpression(node) || isFunctionDeclaration(node)) {
+        return true;
+      }
```

Comment:
Any reason for not performing the fast check within `couldBeFunction`?

Reply:
That's smart, I like it!
To answer your question, I haven't thought of it.